WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/IconListItem.swift

```diff
@@ -10,19 +10,42 @@ struct IconListItem: View {
         /// Base64 icon
         case base64(UIImage)
 
+        case adaptiveBase64(universal: UIImage, dark: UIImage?)
+
         /// Icon that comes from an URL
         case remote(URL)
 
+        /// Adaptive (light/dark mode) remote image, with URLs for one or more parts
+        case adaptiveRemote(universal: URL, dark: URL?)
+
         /// Return an Image for base64 or a FKImage in case of a remote one
         @ViewBuilder
         func getImage() -> some View {
             switch self {
             case .base64(let image):
                 Image(uiImage: image)
                     .resizable()
+            case .adaptiveBase64(let universal, let dark):
+                AdaptiveImage(light: universal, dark: dark)
             case .remote(let url):
                 KFImage(url)
                     .resizable()
+            case .adaptiveRemote(let universal, let dark):
+                AdaptiveAsyncImage(lightUrl: universal, darkUrl: dark, scale: 3) { imagePhase in
```

Comment:
Is hardcoding the scale here to 3 safe enough, or we have to retrieve the scale from an environment variable?

Reply:
Scale is a property of the image we get from the server, not the environment. Right now there's nothing in the What's New Announcement API response which will support a scale, though we could consider using `iconType` for this by using different keys. E.g. `light@3x` `dark@2x`. 

Given that these icons are pretty small, it seemed like a pragmatic solution to assume 3x, as that's big enough for any device.